bin/g

```diff
@@ -556,11 +556,15 @@ open_interactive_ui() {
 # Activate <version>
 #
 
-activate() {
+set_version() {
   version=$1
   active=$(get_current_version)
```

Comment:
@feualpha only another check of `latest` here missing. Otherwise it looks good to merge. :)

Reply:
ah yes, the consequence of exposing it

Reply:
should i make it something like this? since it's already duplicated more then 3 times
```
resolve_version_arg(){
    $version_arg=$1

    if [ "$version_arg" = "latest" ]; then
      echo $(get_latest_version)
    else
      echo $version_arg
    fi
}

version_args=$1
version=$(resolve_version_arg "$version_arg")
```

Reply:
@feualpha ok, let's do it. I usually avoid nesting calls too much since error handling in shell scripting is horrible, but since we are doing command substitution there already, it won't make it worse.

Let's compact it a bit:

```shell
resolve_version_argument() {
    if [ "$1" = "latest" ]; then
      get_latest_version
    else
      echo "$1"
    fi
}
```

And inside the `<x>_version` functions, a single line:

```shell
version=$(resolve_version_argument "$1")
```

Reply:
And @feualpha will need to you to rebase this branch on top of `next` since I just merged #10 which has been in the works for a while now.